workflow/main.jt

```diff
@@ -823,13 +845,61 @@ tools:
 
     input:
       is_allowed:
-        type: tring
+        type: string
       manifest_file:
         type: string
         is_file: true
       song_metadata_url:
         type: string
 
+  create_cgc_manifest:
+    command: |
+      bash -c '
+      docker pull ${util_dckr}
+      docker run -v $(pwd):/data ${util_dckr} \
+        ./generate_cgc_manifest.py \
+        --filenames ${bam_filename} ${bai_filename}
+        --song-payload ${song_payload}
+      echo "{ \"manifest\": "$(pwd)\/manifest.txt" }" > output.json
```

Comment:
commented elsewhere, it should be `echo "{ \"manifest\": \"$(pwd)/manifest.txt\" }" > output.json`